.travis.yml

```diff
@@ -1,11 +1,10 @@
 language: objective-c
 osx_image: xcode10
-xcode_project: AloeStackView.xcodeproj
-xcode_scheme: AloeStackView
-xcode_destination: platform=iOS Simulator,OS=9.0,name=iPhone 6s
 before_script:
     - bundle install
     - brew outdated carthage || brew upgrade carthage
 script:
+    - xcodebuild -project AloeStackView.xcodeproj -scheme AloeStackView -sdk iphonesimulator -destination "platform=iOS Simulator,OS=9.0,name=iPhone 6s" -configuration Debug -PBXBuildsContinueAfterErrors=0 SWIFT_VERSION=4.2 build test
+    - xcodebuild -project AloeStackView.xcodeproj -scheme AloeStackView -sdk iphonesimulator -destination "platform=iOS Simulator,OS=9.0,name=iPhone 6s" -configuration Debug -PBXBuildsContinueAfterErrors=0 SWIFT_VERSION=4.0 build test
```

Comment:
You should run this as two separate parallel jobs, rather than running it twice on the same job.